scheduler/crates/api/src/lib.rs

```diff
@@ -78,7 +78,7 @@ impl Application {
     async fn configure_server(context: NettuContext) -> Result<(Server, u16), std::io::Error> {
         let port = context.config.port;
         let address = format!("127.0.0.1:{}", port);
-        let listener = TcpListener::bind(&address)?;
+        let listener = TcpListener::bind(address)?;
```

Comment:
No need to use a reference, `address` will be converted automatically if needed
https://rust-lang.github.io/rust-clippy/master/index.html#needless_borrows_for_generic_args